build.sbt

```diff
@@ -524,3 +524,55 @@ lazy val fatjarPackageSettings = List[Def.Setting[_]](
     ).transform(node).head
   }
 )
+
+import sbtwelcome._
+
+logo :=
+  raw"""
+       |           _             _                  
+       |          (_)           (_)                 
+       |  ___  ___ _ _ __ ______ _  __ ___   ____ _ 
+       | / __|/ __| | '_ \______| |/ _` \ \ / / _` |
+       | \__ \ (__| | |_) |     | | (_| |\ V / (_| |
+       | |___/\___|_| .__/      | |\__,_| \_/ \__,_|
+       |            | |        _/ |                 
+       |            |_|       |__/                  
+       |
+       |${version.value}
+       |
+       |${scala.Console.YELLOW}Scala ${scalaVersion
+    .value}${scala.Console.RESET}
+       |
+       |""".stripMargin
```

Comment:
Lol, this block of code breaks scalafmt's idempotency, so it's going to be always broken :D 

<img width="898" alt="image" src="https://user-images.githubusercontent.com/1052965/233100785-921bc6c0-8e0a-4899-9bf5-4d7161c8fd9f.png">

@olafurpg I won't risk upgrading scalafmt to a newer version until we don't have any more pressing PRs in flight.

Reply:
Scalafmt is such a fragile formatter can't imagine what ***** wrote this piece of software 🙈

Reply:
:P 

Entire Scala community is still looking for that rapscallion! He's going to _get it_ once we find them.

Reply:
This person is doing a good job hiding in some corner